lib/rules/declaration-block-no-duplicate-properties/index.js

```diff
@@ -85,12 +87,18 @@ const rule = (primary, secondaryOptions) => {
 					return;
 				}
 
-				const indexDuplicate = decls.indexOf(prop.toLowerCase());
+				const indexDuplicate = declsProps.indexOf(prop.toLowerCase());
```

Comment:
[suggest] I notice we can do the following refactorings:

- Avoid repetition of `prop.toLowerCase()`
- Make `declsProps` and `values` variables unneeded by `decls.findIndex()`

For instance,

```js
const lowerProp = prop.toLowerCase();

// ...

const indexDuplicate = decls.findIndex(d => d.prop.toLowerCase() === lowerProp);

// ...

const duplicateDecl = decls[indexDuplicate];
const duplicateValue = duplicateDecl ? duplicateDecl.value : '';
```

Reply:
Also thought of just using decls. Great suggestion!